tools/doc/versions.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
+const https = require('https');
+
 let _versions;
 
 const getUrl = (url) => {
   return new Promise((resolve, reject) => {
-    const https = require('https');
```

Comment:
Regarding this, I'd just like to point out that the change is purely cosmetic just like in the rest of the PR since `http` would be cached anyway...